cves/kernel/CVE-2020-24394.yml

```diff
@@ -75,7 +79,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [962254]
```

Comment:
Can you link the URL to the bug because the bug ID is not found on https://bugzilla.kernel.org/. I'm guessing it is on a different Bugzilla site (